File: app/src/main/java/club/hutcwp/lifeutil/util/DoubleClickExit.java
package club.hutcwp.lifeutil.util;

/**
 * Created by hutcwp on 2017/4/15.
 * Mail : hutcwp@foxmail.com
 * Blog : hutcwp.club
 * GitHub : github.com/hutcwp
 */

public class DoubleClickExit {

    public static long mLastClick = 0L;
    private static final int THRESHOLD = 2000;

    public static boolean check() {
        long now = System.currentTimeMillis();
        boolean b = now - mLastClick < THRESHOLD;
        mLastClick = now;
        return b;
    }
}

File: read/src/main/java/com/hutcwp/read/util/DoubleClickExit.kt
package com.hutcwp.read.util

/**
 * Created by hutcwp on 2017/4/15.
 * Mail : hutcwp@foxmail.com
 * Blog : hutcwp.club
 * GitHub : github.com/hutcwp
 */

object DoubleClickExit {

    var mLastClick = 0L
    private val THRESHOLD = 2000

    fun check(): Boolean {
        val now = System.currentTimeMillis()
        val b = now - mLastClick < THRESHOLD
        mLastClick = now
        return b
    }
}
